main.js

```diff
@@ -177,17 +177,17 @@ const maps = ['gewitter', 'sturm', 'regen', 'schnee', 'nebel', 'frost', 'glattei
 
 function placeWarning(channelName, warnObj) {
     warnObj = warnObj || {};
-    
-    adapter.setForeignState(channelName + '.begin',         tools.formatDate(adapter.formatDate, warnObj.start),  true);
-    adapter.setForeignState(channelName + '.end',           tools.formatDate(adapter.formatDate, warnObj.end),    true);
+    let td = new Date();
+    adapter.setForeignState(channelName + '.begin',         warnObj.start === undefined ? '' : new Date (warnObj.start - td.getTimezoneOffset() * 60000).toISOString(),  true);
```

Comment:
Why you use just `toISOString` instead of `tools.formatDate(adapter.formatDate, date)`?

Reply:
Weil die meisten Zeit-States, die ich bisher gesehen habe das Iso Format ausgeben. Hab mich am häufig aktualisierten Admin-Adapter orientiert und den Datentyp String erhalten.